packages/main/src/themes/sap_horizon/StepInput-parameters.css

```diff
@@ -0,0 +1,24 @@
+@import "../base/StepInput-parameters.css";
+
+:root {
+	--_ui5_step_input_input_error_background_color: inherit;
+	--_ui5-step_input_button_state_hover_backgroun_color: var(--sapField_Hover_Background);
```

Comment:
Here, the typos are two - dash between the ui5 and step and then "backgroun"

Reply:
done